apps/ui/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Snapshot</title>
+    <title></title>
```

Comment:
I think we should still keep Snapshot here by default. (can be changed later) At least for some of the crawlers